tests/testthat/test-meta.R

```diff
@@ -5,7 +5,7 @@ test_that("non-existent file returns default data", {
 test_that("standard metadata is useful", {
   df <- data.frame(x = 1:10)
   path <- withr::local_tempfile()
-  write_rds(df, path)
+  write_rds_test(df, path)
```

Comment:
Here is where I am calling `write_rds_test()`, that function used only here because I can't seem to mock `write_rds()` if I call it directly.